### ISSUE TITLE
Gunners Heaven (Japan) // Sound problem (loop) after selecting a character

### ISSUE THREAD

**Issue by Zet-sensei** [stenzek/duckstation#292]
After selecting one of the two main characters, a sound does a loop during 3-4 seconds before entering the first stage (like a ringing sound, repeating the last sample multiple times).
![Gunners](https://user-images.githubusercontent.com/52127625/79278588-44aec300-7eac-11ea-83d5-7068fe23d2af.jpg)

**Comment by stenzek** [comment]
Sounds like reverb maybe?

**Comment by Zet-sensei** [comment]
The sound isn't from the BIOS, but the last sample from when selecting a character.

The intro FMV, title screen & entering to the selection character screen are fine.

**Comment by stenzek** [comment]
Yeah, I know. I was meaning it in the sense of it may have the same underlying cause, just a different situation.

**Comment by kurayami6** [comment]
Still happens as of the latest version.

[2020-06-04_20-27-07.zip](https://github.com/stenzek/duckstation/files/4729349/2020-06-04_20-27-07.zip)

**Comment by stenzek** [comment]
I think this might just be normal reverb - it's very quiet. If you can get a console recording and it's not present there, I'll re-open.

**Comment by Zet-sensei** [comment]
Recorded some minutes ago from a SCPH-7000 system to a laptop (RCA to 3.5mm microphone jack cable + Audacity 2.3.3 on Recording mode).

[Gunners Heaven.zip](https://github.com/stenzek/duckstation/files/5433063/Gunners.Heaven.zip)

**Comment by stenzek** [comment]
Thanks for the recording! As I said on Discord it's still inconclusive whether the low amplitude reverb signal is there, but I'm leaning towards probably not, so this will require more research.
